scripts/superdesk-authoring/authoring.js

```diff
@@ -384,6 +384,17 @@ import 'angular-history/history.js';
             });
         };
 
+        this.validateBeforeTansa = function (orig, diff) {
```

Comment:
@Aca-jov replied on Slack: the `validate` API endpoint is apparently tansa specific and we already have a private validate function, so we can keep as is.